<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="apples-MacBook-Air.local" name="TestCases.InventoryPageTest" tests="7" failures="1" timestamp="2022-10-20T07:59:37 IST" time="45.906" errors="0">
  <testcase name="verifyBigLogoTest" time="4.568" classname="TestCases.InventoryPageTest"/>
  <system-out/>
  <testcase name="add6ProductTest" time="14.770" classname="TestCases.InventoryPageTest"/>
  <system-out/>
  <testcase name="verifyBigLogoTest" time="5.338" classname="TestCases.InventoryPageTest"/>
  <system-out/>
  <testcase name="verifyBigLogoTest" time="4.977" classname="TestCases.InventoryPageTest"/>
  <system-out/>
  <testcase name="verifySmallLogoTest" time="6.121" classname="TestCases.InventoryPageTest">
    <failure type="java.lang.AssertionError" message="Failed zali ka???????? expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Failed zali ka???????? expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:976)
at TestCases.InventoryPageTest.verifySmallLogoTest(InventoryPageTest.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:44)
at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:72)
at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:10)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- verifySmallLogoTest -->
  <system-out/>
  <testcase name="verifyBigLogoTest" time="5.174" classname="TestCases.InventoryPageTest"/>
  <system-out/>
  <testcase name="verifyBigLogoTest" time="4.958" classname="TestCases.InventoryPageTest"/>
  <system-out/>
</testsuite> <!-- TestCases.InventoryPageTest -->
